owasp/netcdf-java/maint-5.x

```diff
@@ -17,6 +17,16 @@ pipeline {
                 ])
             }
         }
+        stage('Assemble Project') {
+            steps {
+                sh '''
+                   #!/bin/bash
+                   select-java temurin 11
+                   chmod u+x gradlew
+                   ./gradlew assemble
```

Comment:
just for my understanding, why is it necessary to run assemble before the dependency check?

Reply:
That's what I'm finding.  :-|